test/test-lsp-error-reporter.cpp

```diff
@@ -62,6 +62,8 @@ TEST_F(test_lsp_error_reporter, big_int_literal_contains_decimal_point) {
             "BigInt literal contains decimal point");
   EXPECT_EQ(look_up(diagnostics, 0, "code"), "E0005");
   EXPECT_EQ(look_up(diagnostics, 0, "source"), "quick-lint-js");
+  EXPECT_EQ(look_up(diagnostics, 0, "codeDescription", "href"),
+            "https://quick-lint-js.com/errors/E0005");
```

Comment:
This link doesn't work yet. We should update the website first.

Reply:
Should I change the links to the fragment version? Or we update the website later?

Reply:
> Should I change the links to the fragment version?

Yeah.